src/Build/BackEnd/Client/MSBuildClient.cs

```diff
@@ -222,6 +228,16 @@ public MSBuildClientExitResult Execute(string commandLine, CancellationToken can
             return _exitResult;
         }
 
+        private void SupportVT100()
```

Comment:
nit: I think the name should be something like "EnableVT100Mode".

Reply:
I like SupportVT100 a little better, personally. As I understand it, VT100 isn't a mode but characters indicating changes of formatting things.